mongodb/provision.sh

```diff
@@ -33,10 +33,11 @@ install_mongodb() {
         echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu trusty/mongodb-org/3.4 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.4.list
     else
         [ -e /etc/apt/sources.list.d/mongodb-org-3.4.list ] && rm /etc/apt/sources.list.d/mongodb-org-3.4.list
-        if [[ ! $( apt-key list | grep 'MongoDB 4.0') ]]; then
-            apt-key add "${DIR}/server-4.0.asc"
+        [ -e /etc/apt/sources.list.d/mongodb-org-4.0.list ] && rm /etc/apt/sources.list.d/mongodb-org-4.0.list
+        if [[ ! $( apt-key list | grep 'MongoDB 4.4') ]]; then
+            apt-key add "${DIR}/server-4.4.asc"
         fi
-        echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu ${codename}/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
+        echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu ${codename}/mongodb-org/4.4 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.4.list
```

Comment:
the 4.0 file is still going to be there, it'll be joined by a second file for 4.4, but it'll still break when you try to update if you have ever had 4.0 installed

Reply:
I added a line to remove the 4.0 file